src/constants/links.ts

```diff
@@ -7,4 +7,6 @@ export const links: {[name: string]: string} = {
     facebook: 'https://www.facebook.com/FlyByWireSimulations/',
     opencollective: 'https://opencollective.com/flybywire',
     docsfaq: 'https://docs.flybywiresim.com/faq',
+    installer: 'https://cdn.flybywiresim.com/installer/release/FlyByWire_Installer_Setup.exe',
```

Comment:
Can we change this to the CF link?
```suggestion
    installer: 'https://flybywirecdn.com/installer/release/FlyByWire_Installer_Setup.exe',
```